components/brave_rewards/core/endpoints/brave/post_challenges.h

```diff
@@ -0,0 +1,53 @@
+/* Copyright (c) 2024 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_BRAVE_REWARDS_CORE_ENDPOINTS_BRAVE_POST_CHALLENGES_H_
+#define BRAVE_COMPONENTS_BRAVE_REWARDS_CORE_ENDPOINTS_BRAVE_POST_CHALLENGES_H_
+
+#include <string>
+
+#include "base/memory/weak_ptr.h"
+#include "base/types/expected.h"
+#include "brave/components/brave_rewards/common/mojom/rewards.mojom.h"
+#include "brave/components/brave_rewards/common/mojom/rewards_core.mojom.h"
+#include "brave/components/brave_rewards/core/rewards_engine_helper.h"
+
+namespace brave_rewards::internal::endpoints {
+
+// POST /v3/wallet/challenges
+//
+// Request body:
+// {
+//   "paymentId": "<rewards-payment-id>"
+// }
+//
+// Success code: HTTP_CREATED (201)
+//
+// Response body:
+// {
+//   "challengeId": "<challenge-id>"
+// }
+class PostChallenges : public RewardsEngineHelper {
```

Comment:
I'll leave this here for reference: https://github.com/brave/brave-core/commit/1079772a3e650cbd917466f3df98425531bfc2f2